src/main/java/UseCases/FridgeUseCase.java

```diff
@@ -1,48 +1,54 @@
 package UseCases;
 
 import Commands.Implementations.CommandImpl;
-import Storages.Implementations.IngredientStorageImpl;
+import Storages.IngredientStorage;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 
 public class FridgeUseCase implements UseCase {
-    final IngredientStorageImpl ingredientStorage;
-    final IngredientStorageImpl fridge;
+    final IngredientStorage ingredientStorage;
+    final IngredientStorage fridge;
 
     /**
      * Constructor
      */
-    public FridgeUseCase(IngredientStorageImpl fridge, IngredientStorageImpl ingredientStorage) {
+    public FridgeUseCase(IngredientStorage fridge, IngredientStorage ingredientStorage) {
         this.ingredientStorage = ingredientStorage;
         this.fridge = fridge;
     }
 
+    /**
+     * Will either add to fridge or remove from fridge based on what is provided.
+     * NOTE: Command must contain both keys, although neither of them HAVE to contain
```

Comment:
Not true, in fact using `containsKey` means that if the user doesn't want to run a part of the command they can just not even set the key. Remove this line.